ClearKeep/Scenes/TwoFactor/Views/TwoFactorContentView.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  TwoFactorContentView.swift
+//  ClearKeep
+//
+//  Created by đông on 23/03/2022.
+//
+
+import SwiftUI
+import Combine
+import Common
+import CommonUI
+
+private enum Constant {
+	static let spacerTopView = 90.0
+	static let spacer = 25.0
+	static let paddingVertical = 14.0
+	static let cornerRadius = 40.0
+	static let cornerRadiusPasscode = 8.0
+	static let sizePasscodeInput = 90.0
+	static let maxDigits = 4
+}
+
+struct TwoFactorContentView: View {
+	// MARK: - Variables
+	@Environment(\.presentationMode) var presentationMode
+	@Environment(\.injected) private var injected: DIContainer
+	@Environment(\.colorScheme) var colorScheme
+	@Binding private(set) var loadable: Loadable<Bool>
+	@State private var otp: String = ""
+	@State private var otpStyle: TextInputStyle = .default
+	@State private var isToHome: Bool = false
+	@State private(set) var otpHash: String = ""
+	private(set) var userId: String = ""
+	private(set) var domain: String = ""
+	private(set) var password: String = ""
+	let twoFactorType: TwoFactorType
+	let inspection = ViewInspector<Self>()
+	@State var timeRemaining = 120
+	let timer = Timer.publish(every: 1, on: .main, in: .common).autoconnect()
+	@State private var showAlertPopup: Bool = false
+
+	// MARK: - Body
+	var body: some View {
+		content
+			.onReceive(inspection.notice) { inspection.visit(self, $0) }
+			.applyNavigationBarPlainStyle(title: "",
+										  titleColor: AppTheme.shared.colorSet.offWhite,
+										  backgroundColors: backgroundButtonBack,
+										  leftBarItems: {
+				buttonBackView
+			},
+										  rightBarItems: {
+				Spacer()
+			})
+			.onReceive(timer) { _ in
+				if timeRemaining > 0 {
+					timeRemaining -= 1
+					print("\(timeRemaining)")
+				}
+			}
+			.alert("GroupChat.Warning".localized, isPresented: $showAlertPopup) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e5e0a713c2a0f396e41d6374bb8f07759fda2936/ClearKeep/Scenes/TwoFactor/Views/TwoFactorContentView.swift#L61"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>TwoFactorContentView.swift:61</code></td>
    </tr>
  </tbody>
</table>